package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@github/time-elements",
   "version": "3.1.2",
-  "main": "dist/index.js",
+  "main": "dist/bundle.js",
```

Comment:
Just asking, this does not create issues for consumers how do a `require('@github/time-elements')` right? They would still get it, because this looks for whatever is `main` and not for `index.js`?

Reply:
`require()` will key off the `main` key, which defaults to `index.js` - but we've had the `main` key forever so this changes nothing in that regard.

More sophisticated bundlers will look at `module`.